skills/csharp/experimental/itsmskill/Bots/DefaultActivityHandler.cs

```diff
@@ -102,6 +114,18 @@ protected override async Task OnEventActivityAsync(ITurnContext<IEventActivity>
             }
         }
 
+        protected override async Task<InvokeResponse> OnInvokeActivityAsync(ITurnContext<IInvokeActivity> turnContext, CancellationToken cancellationToken)
+        {
+            var itsmTeamsActivityHandler = new ITSMTeamsInvokeActivityHandlerFactory(_serviceProvider);
```

Comment:
i think we should register ITSMTeamsInvokeActivityHandlerFactory within startup as singleton and just use that instance so we don't have multiple instances of it.

Reply:
Added to Startup.cs ....great suggestion